website/docs/r/wafv2_web_acl.html.markdown

```diff
@@ -458,7 +458,7 @@ The `xss_match_statement` block supports the following arguments:
 
 The `excluded_rule` block supports the following arguments:
 
-* `name` - (Required) The name of the rule to exclude.
+* `name` - (Required) The name of the rule to exclude. See the [documentation](https://docs.aws.amazon.com/waf/latest/developerguide/aws-managed-rule-groups-list.html) for a list of names in the "AWSManagedRulesCommonRuleSet" managed rule group.
```

Comment:
since the `excluded_rule` block is also applicable to rule groups managed by practitioners, (i.e. w/in the `rule_group_reference_statement` configuration block), let's make this statement also applicable only to rule groups managed by AWS e.g.
```suggestion
* `name` - (Required) The name of the rule to exclude. If the rule is managed by AWS, see the [documentation](https://docs.aws.amazon.com/waf/latest/developerguide/aws-managed-rule-groups-list.html) for a list of names in the "AWSManagedRulesCommonRuleSet" managed rule group.
```

Reply:
Thanks for the suggestion @anGie44, I've taken your words and expanded on this to be clearer for this use case.

Would you be happy with the changes?

Reply:
looks great @craiggenner , thank you!